swim/node.go

```diff
@@ -66,7 +67,7 @@ func defaultOptions() *Options {
 		RollupFlushInterval: 5000 * time.Millisecond,
 		RollupMaxUpdates:    250,
 
-		Clock: &systemClock{},
+		Clock: nil,
```

Comment:
I think having to init an object here is the cost of using a struct for options. If it is the default option I think we should set it here, not somewhere else. If this is not efficient enough, I think we should consider reworking the design.